src/game.h

```diff
@@ -10,6 +10,7 @@
 #include "item_location.h"
 #include "cursesdef.h"
 #include "ranged.h"
+#include "npc.h"
```

Comment:
You don't need that included. A forward declaration of `npc_template` is enough.